src/AssetPublisher.php

```diff
@@ -557,10 +549,10 @@ private function publishDirectory(string $src, array $options): array
 
         if ($this->linkAssets) {
             if (!is_dir($dstDir)) {
-                FileHelper::createDirectory(\dirname($dstDir), $this->dirMode);
-                try { // fix #6226 symlinking multi threaded
+                FileHelper::createDirectory(dirname($dstDir), $this->dirMode);
+                try { /** fix #6226 symlinking multi threaded */
```

Comment:
It's not a phpdoc. It's bette to use `//` in this case.

Reply:
Fixed.